### PR TITLE
Minor refactoring

### PR DESCRIPTION
**Pull request by nivimsft**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/sql/_params.py [hunk 1]
```diff
@@ -163,6 +163,18 @@ def get_location_type_with_default_from_resource_group(cli_ctx):
     options_list=['--managed-instance', '--mi'],
     help='Name of the Azure SQL managed instance.')
 
+kid_param_type = CLIArgumentType(
+    'kid',
```
Comment (tjprescott) [inline src/azure-cli/azure/cli/command_modules/sql/_params.py]: You need to remove this. 'dest' is not part of a CLIArgumentType.

File: src/azure-cli/azure/cli/command_modules/sql/_params.py [hunk 2]
```diff
@@ -163,6 +163,18 @@ def get_location_type_with_default_from_resource_group(cli_ctx):
...
+    options_list=['--kid', '-k'],
+    help='The Azure Key Vault key identifier of the server key. An example key identifier is '
+    '"https://YourVaultName.vault.azure.net/keys/YourKeyName/01234567890123456789012345678901"')
+
+server_key_type_param_type = CLIArgumentType(
+    'server_key_type',
```
Comment (tjprescott) [inline src/azure-cli/azure/cli/command_modules/sql/_params.py]: Same here.

File: src/azure-cli/azure/cli/command_modules/sql/_params.py (hunk 1 above)
Comment (nivimsft) [reply src/azure-cli/azure/cli/command_modules/sql/_params.py]: addressed

File: src/azure-cli/azure/cli/command_modules/sql/_params.py (hunk 2 above)
Comment (nivimsft) [reply src/azure-cli/azure/cli/command_modules/sql/_params.py]: addressed

**Pull request merged**